examples/actin/docker/src/actin.py

```diff
@@ -76,14 +77,19 @@ def main():
                 "Actin-Polymer",
             )
         ]
-        monomers = ActinGenerator.get_monomers(fiber_data, use_uuids=False)
+        monomers = ActinGenerator.get_monomers(
+            int(parameters["actin_number_types"]), fiber_data, use_uuids=False
```

Comment:
now that we've finished adding the bonds, I'm realizing that we can actually have 5 actin numbers for both with and without longitudinal bonds. The main thing we want to be able to turn on and off is the new bond. There's a little computational overhead from adding the additional constraints and reactions for the extra actin types, but it will also keep the code from getting more complex, e.g. we have to pass `actin_number_types` all around the codebase to the static methods.